<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1467</width>
    <height>865</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #272c36;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_20">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_14">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_12">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="1" column="0">
             <layout class="QGridLayout" name="gridLayout_26">
              <item row="0" column="0">
               <widget class="QPushButton" name="clear_zeros">
                <property name="maximumSize">
                 <size>
                  <width>31</width>
                  <height>31</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">color: white;</string>
                </property>
                <property name="text">
                 <string>del</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QPushButton" name="clear_poles">
                <property name="maximumSize">
                 <size>
                  <width>31</width>
                  <height>31</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">color: white;</string>
                </property>
                <property name="text">
                 <string>del</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="6" column="0">
             <widget class="QPushButton" name="restart_but">
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: white;</string>
              </property>
              <property name="text">
               <string>Restart</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <layout class="QGridLayout" name="gridLayout_27">
              <item row="0" column="0">
               <widget class="QRadioButton" name="zero_button">
                <property name="maximumSize">
                 <size>
                  <width>31</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">color: green;</string>
                </property>
                <property name="text">
                 <string>O</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QRadioButton" name="pole_button">
                <property name="maximumSize">
                 <size>
                  <width>31</width>
                  <height>31</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">color: Red;</string>
                </property>
                <property name="text">
                 <string>X</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="3" column="0">
             <widget class="QCheckBox" name="conjugate">
              <property name="maximumSize">
               <size>
                <width>71</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: white;</string>
              </property>
              <property name="text">
               <string>conjigatet</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="2" column="0">
             <widget class="QPushButton" name="clear_all">
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: white;</string>
              </property>
              <property name="text">
               <string>Clear All</string>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QPushButton" name="load_signal">
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: white;</string>
              </property>
              <property name="text">
               <string>Load_Signal</string>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>upload-square-svgrepo-com.png</normaloff>upload-square-svgrepo-com.png</iconset>
              </property>
             </widget>
            </item>
            <item row="7" column="0">
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item row="0" column="1">
           <layout class="QGridLayout" name="gridLayout_11">
            <item row="0" column="0">
             <layout class="QGridLayout" name="gridLayout_7">
              <item row="0" column="0">
               <widget class="QGroupBox" name="groupBox">
                <property name="maximumSize">
                 <size>
                  <width>400</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <pointsize>9</pointsize>
                  <weight>75</weight>
                  <italic>false</italic>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">color: white;</string>
                </property>
                <property name="title">
                 <string>Z-plane</string>
                </property>
                <layout class="QGridLayout" name="gridLayout">
                 <item row="0" column="0">
                  <widget class="QGraphicsView" name="zplane">
                   <property name="maximumSize">
                    <size>
                     <width>400</width>
                     <height>16777</height>
                    </size>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">color: white;</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item row="0" column="1">
               <layout class="QGridLayout" name="gridLayout_6">
                <item row="0" column="0">
                 <widget class="QGroupBox" name="groupBox_2">
                  <property name="font">
                   <font>
                    <pointsize>9</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">color: white;</string>
                  </property>
                  <property name="title">
                   <string>Magnitude Response </string>
                  </property>
                  <layout class="QGridLayout" name="gridLayout_4">
                   <item row="0" column="2">
                    <widget class="PlotWidget" name="magnitude_response_widget" native="true">
                     <property name="minimumSize">
                      <size>
                       <width>200</width>
                       <height>161</height>
                      </size>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QGroupBox" name="groupBox_3">
                  <property name="font">
                   <font>
                    <pointsize>9</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">color: white;
</string>
                  </property>
                  <property name="title">
                   <string>Phase Response </string>
                  </property>
                  <layout class="QGridLayout" name="gridLayout_5">
                   <item row="0" column="0">
                    <widget class="PlotWidget" name="phase_response_widget" native="true">
                     <property name="minimumSize">
                      <size>
                       <width>200</width>
                       <height>161</height>
                      </size>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item row="1" column="0">
             <layout class="QGridLayout" name="gridLayout_10">
              <item row="0" column="0">
               <widget class="QGroupBox" name="groupBox_6">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>334</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>400</width>
                  <height>16777215</height>
                 </size>
                </property>
                <property name="title">
                 <string/>
                </property>
                <layout class="QGridLayout" name="gridLayout_13">
                 <item row="0" column="0">
                  <widget class="QWidget" name="padd_area" native="true">
                   <property name="minimumSize">
                    <size>
                     <width>376</width>
                     <height>310</height>
                    </size>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">background-color: rgb(0, 0, 0);</string>
                   </property>
                   <layout class="QVBoxLayout" name="verticalLayout_6"/>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item row="0" column="1">
               <layout class="QGridLayout" name="gridLayout_9">
                <item row="1" column="0">
                 <layout class="QGridLayout" name="gridLayout_8">
                  <item row="0" column="0">
                   <spacer name="horizontalSpacer">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                  <item row="0" column="4">
                   <spacer name="horizontalSpacer_3">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>40</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                  <item row="0" column="2">
                   <widget class="QSlider" name="Speed_slider">
                    <property name="maximumSize">
                     <size>
                      <width>16777215</width>
                      <height>22</height>
                     </size>
                    </property>
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <spacer name="horizontalSpacer_2">
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                    <property name="sizeType">
                     <enum>QSizePolicy::Preferred</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>60</width>
                      <height>20</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                  <item row="0" column="3">
                   <widget class="QLabel" name="speed_val">
                    <property name="styleSheet">
                     <string notr="true">color: white;
</string>
                    </property>
                    <property name="text">
                     <string>1 points/sec</string>
                    </property>
                    <property name="alignment">
                     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item row="0" column="0">
                 <layout class="QGridLayout" name="gridLayout_2">
                  <item row="0" column="0">
                   <widget class="QGroupBox" name="groupBox_7">
                    <property name="font">
                     <font>
                      <pointsize>9</pointsize>
                      <weight>75</weight>
                      <bold>true</bold>
                     </font>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">color: white;</string>
                    </property>
                    <property name="title">
                     <string>Real Time Signal </string>
                    </property>
                    <layout class="QGridLayout" name="gridLayout_15">
                     <item row="0" column="0">
                      <widget class="PlotWidget" name="input_signal_graph" native="true">
                       <property name="minimumSize">
                        <size>
                         <width>200</width>
                         <height>161</height>
                        </size>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QGroupBox" name="groupBox_8">
                    <property name="font">
                     <font>
                      <pointsize>9</pointsize>
                      <weight>75</weight>
                      <bold>true</bold>
                     </font>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">color: white;</string>
                    </property>
                    <property name="title">
                     <string>Filtered Signal </string>
                    </property>
                    <layout class="QGridLayout" name="gridLayout_16">
                     <item row="0" column="0">
                      <widget class="PlotWidget" name="output_signal_graph" native="true">
                       <property name="minimumSize">
                        <size>
                         <width>200</width>
                         <height>161</height>
                        </size>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_21">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_19">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_17">
            <item row="1" column="0">
             <widget class="QLabel" name="label">
              <property name="maximumSize">
               <size>
                <width>256</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">color: white;</string>
              </property>
              <property name="text">
               <string>All Pass Filter</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QListWidget" name="filtersList">
              <property name="maximumSize">
               <size>
                <width>600</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <layout class="QGridLayout" name="gridLayout_18">
              <item row="1" column="0">
               <layout class="QFormLayout" name="formLayout_2">
                <item row="0" column="0">
                 <widget class="QPushButton" name="add_filter_but">
                  <property name="maximumSize">
                   <size>
                    <width>93</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">color: white;</string>
                  </property>
                  <property name="text">
                   <string>Add</string>
                  </property>
                  <property name="shortcut">
                   <string notr="true">Return</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="0" column="0">
               <layout class="QFormLayout" name="formLayout">
                <item row="0" column="0">
                 <widget class="QLabel" name="label_2">
                  <property name="styleSheet">
                   <string notr="true">color: white;
</string>
                  </property>
                  <property name="text">
                   <string>A Value</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QLineEdit" name="lineEdit">
                  <property name="maximumSize">
                   <size>
                    <width>203</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">color: white;
</string>
                  </property>
                  <property name="inputMethodHints">
                   <set>Qt::ImhNone</set>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="clearButtonEnabled">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item row="0" column="1">
           <layout class="QGridLayout" name="gridLayout_24">
            <item row="0" column="0">
             <widget class="QGroupBox" name="groupBox_9">
              <property name="minimumSize">
               <size>
                <width>1200</width>
                <height>0</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>9</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: white;</string>
              </property>
              <property name="title">
               <string>Selected Filter Phase</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_22">
               <item row="0" column="0">
                <widget class="PlotWidget" name="selescte_filter_phase" native="true">
                 <property name="minimumSize">
                  <size>
                   <width>900</width>
                   <height>161</height>
                  </size>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QGroupBox" name="groupBox_10">
              <property name="minimumSize">
               <size>
                <width>1000</width>
                <height>0</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>9</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: white;</string>
              </property>
              <property name="title">
               <string>All_pass_phase</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_23">
               <item row="0" column="0">
                <widget class="PlotWidget" name="all_pass_phase" native="true">
                 <property name="minimumSize">
                  <size>
                   <width>900</width>
                   <height>161</height>
                  </size>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item row="0" column="1">
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Preferred</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>50</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1467</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header location="global">pyqtgraph</header>
   <container>1</container>
   <slots>
    <signal>signal1()</signal>
    <slot>slot1()</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
